inscriptions/ISic003684.xml

```diff
@@ -191,20 +196,16 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140929">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AL">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— —</supplied>
-                    Βελλ(—) ∙ Ἑρμῆς
-                    <supplied reason="lost">— —</supplied>
-                    <lb n="2"/>
-                    <supplied reason="lost">— —</supplied>ΤΩΑΡΧ
-                    <supplied reason="lost">— — — — — —</supplied>
-                    <lb n="3"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> <orig>ΒΕΛΛ</orig> <g ref="#interpunct">·</g> <w>Ἑρμ<hi rend="ligature">ῆς</hi></w> <gap reason="lost" extent="unknown" unit="character"/>
```

Comment:
hermes is presumably a name (even if at this point it simply is given a name tag and nothing more)